lib/ember-test-helpers/test-module.js

```diff
@@ -213,7 +214,9 @@ export default Klass.extend({
 
 
   _setupIsolatedContainer: function() {
-    this.container = isolatedContainer(this.needs);
+    var isolated = isolatedContainer(this.needs);
+    this.container = isolated.container;
+    this.registry = isolated.registry;
```

Comment:
This makes the isolated case consistent with the integrated case, which already provides `this.registry`.